services/api_service.ts

```diff
@@ -16,16 +16,10 @@ enum URL {
 
 class APIImpl implements APIService {
   private COHERE_API_KEY = "";
-  private COHERE_VERSION = "";
+  private COHERE_VERSION = "/v1";
```

Comment:
would prefer it if the version was just `1` and the path would be `v{this.COHERE_VERSION}/endpoint`